docs/docs/cmd/spo/folder/folder-list.md

```diff
@@ -25,3 +28,99 @@ Gets list of folders under a parent folder with site-relative url _/Shared Docum
 ```sh
 m365 spo folder list --webUrl https://contoso.sharepoint.com/sites/project-x --parentFolderUrl '/Shared Documents'
 ```
+
+Gets recursive list of folders under a specific folder on a specific site
+
+```sh
+m365 spo folder list --webUrl https://contoso.sharepoint.com/sites/project-x --parentFolderUrl '/Shared Documents' --recursive
+```
+
+## Response
+
+### Standard response
+
+=== "JSON"
+
+    ``` json
+    [  
+      {
+        "Exists": true,
+        "IsWOPIEnabled": false,
+        "ItemCount": 9,
+        "Name": "Folder A",
+        "ProgID": null,
+        "ServerRelativeUrl": "/Shared Documents/Folder A",
+        "TimeCreated": "2022-04-26T12:30:56Z",
+        "TimeLastModified": "2022-04-26T12:50:14Z",
+        "UniqueId": "20523746-971b-4488-aa6d-b45d645f61c5",
+        "WelcomePage": ""
+      }
+    ]
+    ```
+
+=== "Text"
+
+    ``` text
+    Name     ServerRelativeUrl
+    -------  -------------------------
+    Folder A /Shared Documents/Folder A
+    ```
+
+=== "CSV"
+
+    ``` CSV
+    Name,ServerRelativeUrl
+    Folder A,/Shared Documents/Folder A
+    ```
+
+### `recursive` response
```

Comment:
Let's remove this section from the docs. The objects returned are exactly the same as the one above, also we require that only 1 object is shown so it doesn't take too much space on our docs.